test/SemVer.unittest.js

```diff
@@ -57,6 +72,64 @@ describe("SemVer", () => {
 		]);
 	});
 
+	it("should parseVersion correctly (runtime)", () => {
+		const parseVersionRuntime = createRuntimeFunction(parseVersionRuntimeCode);
```

Comment:
Could you avoid duplicating the test by using e.g.

```js
for (const [name, fn] of [["normal", parseVersion], ["runtime", parseVersionRuntime]])
```

Reply:
Could do. I considered it but thought the tests that are already quite loopy would get harder to follow what's going on.

Happy to update if you prefer.

Reply:
Done